Library/Homebrew/rubocops/cask/on_system_conditionals.rb

```diff
@@ -1,4 +1,4 @@
-# typed: true
+# typed: false
```

Comment:
Do we want to add this back eventually?

Reply:
Yeah, may as well. I changed it because `def_node_matcher` and `def_node_search` mean that we need to add an rbi file which I was just avoiding since the rubocops aren't really user-facing, but I guess there's no reason not to.